ci/scripts/rust_build.sh

```diff
@@ -21,6 +21,10 @@ set -ex
 
 source_dir=${1}/rust
 
+# This file is used to build the rust binaries needed for the
```

Comment:
Doing a full rust build as part of the integration test appears to be unecessary